dbus/src/channel/ffichannel.rs

```diff
@@ -351,7 +350,7 @@ impl Channel {
 
 impl Watch {
     unsafe fn from_raw_enabled(watch: *mut ffi::DBusWatch) -> (Self, bool) {
-        let mut w = Watch { fd: ffi::dbus_watch_get_unix_fd(watch), read: false, write: false};
+        let mut w = Watch { fd: ffi::dbus_watch_get_unix_fd(watch) as Fd, read: false, write: false};
```

Comment:
According to D-Bus docs for dbus_watch_get_unix_fd():

Always returns -1 on Windows. On Windows you use dbus_watch_get_socket() to get a Winsock socket to watch.

Reply:
Actually, on Windows `dbus_watch_get_unix_fd()` returns what `dbus_watch_get_socket()` returns. Though yes, better to use them according to the docs